trunner/runners/Stm32l4Runner.py

```diff
@@ -0,0 +1,77 @@
+#
+# Phoenix-RTOS test runner
+#
+# Stm32l4 runner
+#
+# Copyright 2021 Phoenix SYstems
+# Authors: Mateusz Niewiadomski
+#
+import subprocess
+import serial
+import time
+import pexpect
+import pexpect.fdpexpect
+
+from .common import DeviceRunner
+from .common import _BOOT_DIR
+
+
+class Stm32l4Runner(DeviceRunner):
```

Comment:
maybe `STM32L4Runner` as lowercase `l` looks like `1`

Reply:
Good advice. Changed target name to uppercase (only not in ` message where there is a convention of full lowercase). Thanks!